engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToHMDNode.java

```diff
@@ -117,5 +119,14 @@ private void renderFinalStereoImage(RenderingStage renderingStage) {
                 GL11.glFinish();
                 break;
         }
+
+        // Bind the default FBO. The DAG does not recognize that this node has
+        // bound a different FBO, so as far as it is concerned, FBO 0 is still
+        // bound. As a result, without the below line, the image is only copied
+        // to the HMD - not to the screen as we would like. To get around this,
+        // we bind the default FBO here at the end.  This is a bit brittle
+        // because it assumes that FBO 0 is bound before this node is run.
+        // TODO: break this node into two different nodes that use addDesiredStateChange(BindFBO...))
+        new BindFBOTask(0,DEFAULT_FRAME_BUFFER_URN).execute();
```

Comment:
2. I added a TODO.

3. As I understand ASW, it uses the depth buffer to reproject results from the previous frame into the missed frame using the tracking data. By knowing the position of a particle at time `t_1` and the head pose at time `t_2`, it's possible to project that particle into the reference frame needed for time `t_2`. If there were no depth buffer provided, I'd guess that the SDK just wouldn't be able to do that.
Once again, just speculation here. In order to get ASW working I would have to sit down and read about it, as well as verify it were working through experimentation.